frontend/src/layouts/MainLayout.vue

```diff
@@ -346,611 +346,585 @@
-        this.reconnectLoop = null
-      }
-      if (this.flags.autoReconnect) {
-        this.reconnectLoop = setInterval(async () => {
-          if (this.flags.autoReconnect) {
-            const ports = await this.findKnownDevices()
-            if (ports && ports.length > 0) {
-              clearInterval(this.reconnectLoop)
-              this.reconnectLoop = null
-              return await this.start()
-            }
-          } else {
-            clearInterval(this.reconnectLoop)
-            this.reconnectLoop = null
-          }
-        }, 3000)
-      }
-    },
-
-    toggleConnectOnStart () {
-      localStorage.setItem('connectOnStart', this.flags.connectOnStart)
-    },
-    toggleAutoReconnect () {
-      localStorage.setItem('autoReconnect', this.flags.autoReconnect)
-    },
-    toggleInstallFromFile () {
-      localStorage.setItem('installFromFile', this.flags.installFromFile)
-    },
-    toggleCatalogEnabled () {
-      localStorage.setItem('catalogEnabled', this.flags.catalogEnabled)
-      location.reload()
-    },
-    toggleCatalogChannel () {
-      localStorage.setItem('catalogChannel', this.flags.catalogChannelProduction ? 'production' : 'dev')
-      location.reload()
-    },
-    setRpcStatus (s) {
-      this.flags.rpcActive = s
-    },
-    setInfo (info) {
-      this.info = info
-    },
-    onUpdateStage (stage) {
-      if (stage === 'start') {
-        this.flags.updateInProgress = true
-      } else if (stage === 'end') {
-        this.flags.updateInProgress = false
-      }
-    },
-    openFileIn ({ path, file }) {
-      this.log({
-        level: 'info',
-        message: `${this.componentName}: Passing file ${file.name} to ${path}`
       })
-      this.fileToPass = file
-      this.$router.push(path)
-    },
-
-    checkConnectionRequirement (path) {
-      this.flags.connectionRequired = true
-      for (const link of this.canLoadWithoutFlipper) {
-        if ((path && path.includes(link)) || location.pathname.includes(link)) {
-          this.flags.connectionRequired = false
-          break
-        }
-      }
+      .catch(error => rpcErrorHandler(error, 'storageInfo'))
+    info.value.storage.sdcard.status = 'installed'
+    info.value.storage.sdcard.totalSpace = extInfo.totalSpace
+    info.value.storage.sdcard.freeSpace = extInfo.freeSpace
+  } else {
+    info.value.storage.sdcard.status = 'missing'
+    info.value.storage.databases.status = 'missing'
+  }
 
-      if (this.flags.catalogEnabled !== true) {
-        this.routes = this.routes.filter(e => e.link !== '/apps')
-        if (location.pathname === '/apps') {
-          this.$router.push('/')
+  const intInfo = await flipper.RPC('storageInfo', { path: '/int' })
+    .then(() => {
+      log({
+        level: 'debug',
+        message: `${componentName}: storageInfo: /int`
+      })
+    })
+    .catch(error => rpcErrorHandler(error, 'storageInfo'))
+  info.value.storage.internal.totalSpace = intInfo.totalSpace
```

Comment:
Error in the console. Error in the console. Therefore, it is impossible to check the pages where Flipper is required

<img width="650" alt="image" src="https://github.com/flipperdevices/lab.flipper.net/assets/52247631/4268a6f6-deab-449d-a9bb-6208d67ad129">